<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigTerrain</class>
 <widget class="QDialog" name="ConfigTerrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Controle de Atualização de Terreno</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeDate">
         <property name="minimumSize">
          <size>
           <width>123</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Terrenos</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblTerrainSelected">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Data selecionada</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="butFile">
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Busca local de terreno</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butFtpConn">
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Conecta ao servidor de terrenos</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butDelete">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Exclui terreno na data selecionada</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butExport">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Exclui terreno na data selecionada</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>1</number>
          </property>
          <item>
           <widget class="QCalendarWidget" name="calendar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="gridVisible">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QCalendarWidget::SingleSelection</enum>
            </property>
            <property name="horizontalHeaderFormat">
             <enum>QCalendarWidget::ShortDayNames</enum>
            </property>
            <property name="verticalHeaderFormat">
             <enum>QCalendarWidget::NoVerticalHeader</enum>
            </property>
            <property name="navigationBarVisible">
             <bool>true</bool>
            </property>
            <property name="dateEditEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableDescription">
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nome</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Registro do sistema</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Notas</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="butConfirm">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>61</width>
           <height>23</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Confirma alteração de terreno</string>
         </property>
         <property name="text">
          <string>Carregar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butClose">
         <property name="minimumSize">
          <size>
           <width>46</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>46</width>
           <height>23</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Fecha janela de controle de configuração de terreno</string>
         </property>
         <property name="text">
          <string>Fechar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeDate</tabstop>
  <tabstop>butFtpConn</tabstop>
  <tabstop>butFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
